mypyc/genops.py

```diff
@@ -967,7 +976,7 @@ def visit_mypy_file(self, mypyfile: MypyFile) -> None:
 
         classes = [node for node in mypyfile.defs if isinstance(node, ClassDef)]
 
-        # Collect all classes.
+        # Collect all undecorated classes.
```

Comment:
This is inaccurate, right? It collects decorated ones also?

(Would it be better if we didn't bother generating class_irs for decorated ones? No idea; I might be wrong here.)